### ISSUE TITLE
Missing basic accessibility features

### ISSUE THREAD

**Issue by maxchernin** [dreyescat/react-rating#101]
Hi guys,
acessability should support atleast two main features,
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
and
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

does this component support inputProps to control the native element?
its recommended to add this attr to accept an object and apply all native attrs like tabindex passed in this obj.
can you replace the rating symbol component with a interactive element instead of the elment?
this would make it possible to navigate on each symbol using the tab button

aria-label passed to inputprops will allow this component (the rating - not rating symbol) to read a message to the user.

Please consider adding these two first, especially the tabindex, as it would make this component work well on forms, and overall

**Comment by dreyescat** [comment]
> does this component support inputProps to control the native element?

In #96 we propagated some properties to the wrapper element. Do you mean propagating `tabindex` and `aria-label` too?

> can you replace the rating symbol component with a interactive element instead of the elment?

Uhm, not sure what you mean with *interactive element*. Do you mean `input` for example?

**Comment by maxchernin** [comment]
> In #96 we propagated some properties to the wrapper element. Do you mean propagating tabindex and aria-label too?
Yes, exactly that

> Uhm, not sure what you mean with interactive element. Do you mean input for example?

right now the rendered text is inside a <span> element
read more here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Interactive_elements
its not as important as adding the tabindex.
the preferred behaviour would be such as the displayed here: 
https://material-ui.com/demos/selection-controls/
https://codesandbox.io/s/pp99j7zq8q
play with the tabs and see for yourself, this code doesn't contain tabindex because the input element will be focused by tab key, depending on the render oreder, but for now i think adding aria+tabindex would be enough

**Comment by dreyescat** [comment]
Both attributes (`tabIndex` and `aria-label`) are propagated to the wrapper element.

Released in version [1.3.0](https://github.com/dreyescat/react-rating/releases/tag/v1.3.0).

**Comment by dwoodwardgb** [comment]
While it is nice to have a tabIndex, and aria-label, there is still not a way to select one of the values using a keyboard.

Am I missing something?

**Comment by dreyescat** [comment]
You are right @dwoodwardgb. This issue was just to let `tabIndex` and `aria-label` properties be propagated to the top level element. It was just a workaround to allow at least these *basic* accessibility properties.

For a more general accessibility issue we have #61. When this component was created I didn't even think about accessibility so to add it now I guess it will need some kind of refactor. In that issue there are at least a couple of proposals, as input range and as radio buttons. Any thought and help is really welcome.
